src/style.css

```diff
@@ -2,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  box-sizing: border-box;
+  padding-inline-start: 0;
+  list-style-type: none;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
```

Comment:
Make sure that the path to the image is correct. The current path suggests that the image is located in an `images` folder, but according to the project structure, it should be in the `stars` folder.